calm/draft/2024-10/meta/flow.json

```diff
@@ -0,0 +1,82 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://calm.finos.org/draft/2024-10/meta/flow.json",
+  "title": "Business Flow Model",
+  "description": "Defines business flows that relate to technical architectures, allowing mapping of flows to technical components and attaching control requirements.",
+  "type": "object",
+  "properties": {
+    "unique-id": {
+      "type": "string",
+      "description": "Unique identifier for the flow"
+    },
+    "name": {
+      "type": "string",
+      "description": "Descriptive name for the business flow"
+    },
+    "description": {
+      "type": "string",
+      "description": "Detailed description of the flow's purpose"
+    },
+    "requirement-url": {
+      "type": "string",
+      "description": "Link to a detailed requirement document"
+    },
+    "transitions": {
+      "type": "array",
+      "items": {
+        "$ref": "#/defs/transition"
+      }
+    },
+    "controls": {
+      "$ref": "https://calm.finos.org/draft/2024-10/meta/control.json#/defs/controls"
+    },
+    "metadata": {
+      "$ref": "#/defs/metadata"
+    }
+  },
+  "required": [
+    "unique-id",
+    "name",
+    "description",
+    "transitions"
+  ],
+  "defs": {
+    "transition": {
+      "type": "object",
+      "properties": {
+        "relationship-unique-id": {
+          "type": "string",
+          "description": "Unique identifier for the relationship in the architecture"
+        },
+        "sequence-number": {
+          "type": "integer",
```

Comment:
Just want to check we're happy with the need for sequence number, what do we see the value of this over simply the ordering of the array?

Reply:
See discussion https://github.com/finos/architecture-as-code/pull/463#discussion_r1795353370